src/sentry/issues/ignored.py

```diff
@@ -19,6 +29,33 @@ def handle_archived_until_escalating(
     metrics.incr("group.archived_until_escalating", skip_internal=True)
     for group in group_list:
         remove_group_from_inbox(group, action=GroupInboxRemoveAction.IGNORED, user=acting_user)
-    # TODO(snigdha): create a forecast for this group
+
+    forecasts = get_forecasts(list(group_list))
```

Comment:
With regards to this block of code.

A lot of it is very similar to https://github.com/getsentry/sentry/blob/49b3a40f84c11f0b819c58b8ad932a4e6a300682/src/sentry/tasks/weekly_escalating_forecast.py#L47-L64

@jangjodi when you switch to nodestore. Could you also refactor both pieces of code to be using the same code?

Perhaps we put it all *common* code within src/sentry/issues/escalating.py instead of the weekly task.

Reply:
My apologies, didn't see this until now. Here is the [PR](https://github.com/getsentry/sentry/pull/47586) for it